### ISSUE TITLE
elfinder button names

### ISSUE THREAD

**Issue by achezzoh** [nao-pon/elFinder-nightly#12]
i have a small issue with elfinder 2.1. the command buttons have very unfriendly titles when hovered eg "rm" for the delete button. is it possible to change this anywhere? also the confirmation dialog for delete has the same unfriendly labels for the confirm and cancel buttons. please help.

**Comment by nao-pon** [comment]
@achezzoh Try include "elfinder.en.js" for fall back.

This is my demo site's elfinder.html
```html
<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>elFinder 2.1</title>
		<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2" />

		<!-- jQuery and jQuery UI (REQUIRED) -->
		<link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/smoothness/jquery-ui.css">
		<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
		<script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script>

		<!-- elFinder CSS (REQUIRED) -->
		<link rel="stylesheet" type="text/css" href="css/elfinder.min.css">
		<link rel="stylesheet" type="text/css" href="css/theme.css">

		<!-- elFinder JS (REQUIRED) -->
		<script src="js/elfinder.min.js"></script>

		<!-- elFinder translation fall back -->
		<script src="js/i18n/elfinder.en.js"></script>

		<!-- elFinder initialization (REQUIRED) -->
		<script type="text/javascript" charset="utf-8">
			// Documentation for client options:
			// https://github.com/Studio-42/elFinder/wiki/Client-configuration-options
			$().ready(function() {

				var getLang = function() {
					try {
						var full_lng = (navigator.browserLanguage || navigator.language || navigator.userLanguage);
						var lng = full_lng.substr(0,2);
						if (lng == 'ja') lng = 'jp';
						else if (lng == 'pt') lng = 'pt_BR';
						else if (lng == 'zh') lng = (full_lng.substr(0,5) == 'zh-tw')? 'zh_TW' : 'zh_CN';

						if (lng != 'en') {
							var script_tag		= document.createElement("script");
							script_tag.type		= "text/javascript";
							script_tag.src		= "js/i18n/elfinder."+lng+".js";
							script_tag.charset = "utf-8";
							$("head").append(script_tag);
						}

						return lng;
					} catch(e) {
						return 'en';
					}
				};

				var elfinderInstance = $('#elfinder').elfinder({
					height: $(window).height() - 20,
					ui  : ['toolbar', 'places', 'tree', 'path', 'stat'],
					url : 'php/connector.minimal.php',  // connector URL (REQUIRED)
					lang: getLang()                     // language (OPTIONAL)
				}).elfinder('instance');

				// set document.title dynamically etc.
				var title = document.title;
				elfinderInstance.bind('open', function(event) {
					var data = event.data || null;
					var path = '';
					
					if (data && data.cwd) {
						path = elfinderInstance.path(data.cwd.hash) || null;
					}
					document.title =  path? path + ':' + title : title;
				});

				// fit to window.height on window.resize
				var resizeTimer = null;
				$(window).resize(function() {
					resizeTimer && clearTimeout(resizeTimer);
					resizeTimer = setTimeout(function() {
						var h = parseInt($(window).height()) - 20;
						if (h != parseInt($('#elfinder').height())) {
							elfinderInstance.resize('100%', h);
						}
					}, 200);
				});

			});
		</script>
	</head>
	<body>

		<!-- Element where elFinder will be created (REQUIRED) -->
		<div id="elfinder"></div>

	</body>
</html>
```

**Comment by achezzoh** [comment]
thanks a lot, it worked!

**Comment by nao-pon** [comment]
:+1:

**Comment by nao-pon** [comment]
@achezzoh I made mistake in Jakefile.js (Build config). But it was fixed. Please try latest nightly. And You should remove "elfinder.en.js". That is included in "elfinder.min.js" of "elfinder.full.js".

**Comment by achezzoh** [comment]
I had not even noticed it-the problem with being just a user! Anyway thanks again, leme check i out.
